.eslintrc.js

```diff
@@ -1,58 +1,60 @@
+'use strict';
+
 module.exports = {
   root: true,
   parser: 'babel-eslint',
   parserOptions: {
     ecmaVersion: 2018,
-    sourceType: 'module'
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
   },
-  plugins: ['ember', 'prettier'],
```

Comment:
don't we still need the prettier plugin?

https://github.com/kturney/ember-mapbox-gl/blob/master/.eslintrc.js